moviebot/nlu/annotation/joint_bert/joint_bert_train.py

```diff
@@ -22,7 +22,7 @@
 Batch = Tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]
 
 _MODEL_OUTPUT_PATH = "models/joint_bert"
-_DATA_PATH = "data/training/nlu/utterances.yaml"
+_DATA_PATH = "data/training/utterances.yaml"
 
 _IGNORE_INDEX = -100
```

Comment:
It looks like the same variable is defined in `dataset.py`, from my understanding these need to have the same value. So, I would suggest to keep only one and import it in the otehr file.

Reply:
Done.